### DISCUSSION TITLE
Execute on container readiness

### DISCUSSION THREAD

**Discussion by lohmag** [flant/shell-operator#96] [Q&A]
I can't find a way to execute after container readiness is true.
It can be done with - 
`jq -r 'select(.status.containerStatuses[].ready==true)'`
Is it possible?

**Comment by diafour** [comment]
Hello! Did you try this expression as `jqFilter` and what happened? Can you show a hook configuration and what version you use, `v1.0.0-beta.5` or `latest`?

**Comment by dmgtn** [comment]
@diafour, maybe we add such case to the examples?
